### PR TITLE
Added aggregation of data for index page and test

### LINKED ISSUE #7: Aggregate data for index page
Follow up of issue #3 

# Problem

Loading data in the main view of the `content` app doesn't aggregate data:

https://github.com/vrde/letscheck/blob/47f357681b8955bbb30291cd986036036a4a3a3c/src/content/templates/content/index.html#L97

This is `O(n)` because it generates a new query for each entry and it doesn't use the nice aggregate functions of SQL.

# Solution

I'm a bit rusty with Django, but [`annotate`](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#annotate) should do the job.

# Acceptance

Make sure (using the django debug toolbar) that the number of queries is not `O(n)`, where `n` is the number of news. Add a test that creates a news, attaches multiple cases to it, and verifies that the number displayed in the page is correct.

### PR DESCRIPTION
**Pull request by lorrandal**
Fix #7 

If this part looks too verbose,

https://github.com/lorrandal/letscheck/blob/0d30d614c2aa73ef2d671a747253ad4d0e4973c8/src/content/tests.py#L51-L54

I could change in:

`Case.objects.filter(pk=message_1.pk).update(response=message_2, news=news)`

### CODE REVIEW

File: src/content/views.py [hunk 1]
```diff
@@ -11,7 +12,7 @@
 
 
 def index(request):
-    all_news = News.objects.select_related('rating')
+    all_news = News.objects.select_related('rating').annotate(cases=Count('case'))
```
Comment (vrde) [inline src/content/views.py]: Changing `cases` to `cases_count` makes it easier to guess what the property is about. `cases` is too generic and can be mistaken for something similar to `case_set`.

**Review by vrde — CHANGES_REQUESTED** [review]
Great! I have one simple change to request, take a look :clap:

File: src/content/views.py (hunk 1 above)
Comment (lorrandal) [reply src/content/views.py]: Sure it was quite ambiguous, now it's fixed!

**Review by vrde — APPROVED** [review]
Perfect, thanks!

**Pull request merged**